src/Exception/ProxyResponseException.php

```diff
@@ -11,32 +11,27 @@
 
 namespace FOS\HttpCache\Exception;
 
+use Http\Adapter\Exception\HttpAdapterException;
```

Comment:
No, this is the HTTP client adapter package that we use: https://github.com/php-http. Remember that PSR7 is only about HTTP *messages*.